terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/logging/LoggingTest.java

```diff
@@ -50,7 +50,7 @@ public void test01_01_createDefaultXTrackMDC() {
 
         // cut x-Track MDC
         String targetMdc = driver.findElement(By.id("xTrackMDC")).getText();
-        String footerMdc = driver.findElement(By.cssSelector("p")).getText();
+        String footerMdc = driver.findElement(By.id("xtrack")).getText();

```

Comment:
@kazuki43zoo 
I think...
This logic is to treat common view item, good idea.
But given that various developers copy and use `WebDriverOperations` etc, it's not good to add a method to get elements that [aren't exist default in our blank project](https://github.com/terasolunaorg/terasoluna-gfw-web-multi-blank/blob/master/projectName-web/src/main/webapp/WEB-INF/views/layout/template.jsp#L27-L28). (However, with this idea, `WebDriverOperations` should not hold `getApServerName` method added this time...)
How is it?

I think also an option to add another `WebDriverOperations` etc to our blank project.